ubiquibot-config-default.yml

```diff
@@ -0,0 +1,41 @@
+---
+chain-id: 1
+base-multiplier: 1000
+issue-creator-multiplier: 2000
+time-labels:
+  - name: "Time: <1 Hour"
+    weight: 0.125
+    value: 3600
+  - name: "Time: <1 Day"
+    weight: 1
+    value: 86400
+  - name: "Time: <1 Week"
+    weight: 2
+    value: 604800
+  - name: "Time: <2 Weeks"
+    weight: 3
+    value: 1209600
+  - name: "Time: <1 Month"
+    weight: 4
+    value: 2592000
```

Comment:
Value should be dynamically parsed from the time value using the `ms` library as we already discussed in the past @0xcodercrane

Reply:
@0xcodercrane was that ever implemented? I vaguely recall reviewing the pull request for it.

Reply:
it has not been implemented even on this PR.

Reply:
Okay #431 was filed.